README.md

```diff
@@ -243,6 +243,7 @@ This selector is applicable when "deviceType" is "netDevice"(note: this is defau
 |---------------|----------|----------------------------------------------------------------|---------------------------------------------------|--------------------------------------------------------------------------------------|
 | "pfNames"     | N        | VFs from PF matches list of PF names                           | `string` list Default: `null`                     | "pfNames": ["enp2s2f0"] (See follow-up sections for some advance usage of "pfNames")             |
 | "rootDevices"     | N        | VFs from PF matches list of PF PCI addresses               | `string` list Default: `null`                     | "rootDevices": ["0000:86:00.0"] (See follow-up sections for some advance usage of "rootDevices") |
+| "nicNames"    | N        | VFs from PF matches list of interface names                    | `string` list Default: `null`                     | "nicNames": ["net1"]                                                                             |
```

Comment:
I think the description is not right here. the nic doesn't need to be a VF from a PF it can be a regular nic. special if you run this device plugin inside a VM there is no PF supporter nic